Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/trees/Tree.tsx

```diff
@@ -14,11 +14,12 @@ import styles from './Tree.module.scss';
 
 interface ITreeProps {
     items: ITreeItemType[];
-    onSelect: (node: ITreeItemType) => void;
+    onSelect: (node: any) => void;
     defaultSelected?: string;
     defaultExpanded?: string[];
     itemFunc?: (item: ITreeItemType) => React.ReactElement;
     treeItemHasTooltip?: boolean;
+    onTreeItemClick?: (id: string, node: any) => void;
```

Comment:
What is the difference of `onSelect` and `onTreeItemClick`?

This component, `Tree`, should not have 2 click events